tables/isolates.csv

```diff
@@ -1197,4 +1197,6 @@ hCoV-19/Hong Kong/HKU-344/2021,Omicron,FALSE,7357684,NULL,TRUE
 hCoV-19/Japan/UT-NCGM02/2020,A,FALSE,418809,NULL,TRUE
 hCoV-19/Russia/MOW-Moscow_PMVL-O16/2021,Omicron,FALSE,7263933,NULL,TRUE
 hCoV-19/USA/GA-EHC-2811C/2021,Omicron,FALSE,7171744,NULL,TRUE
-B.1.2 full genome,B.1.2,FALSE,NULL,NULL,TRUE
+Chen21e-B.1.1.7,B.1.2,FALSE,NULL,NULL,TRUE
+WA1+S:614G,NULL,TRUE,NULL,NULL,TRUE
+WA1+S:501Y+614G,NULL,FALSE,NULL,NULL,TRUE

```

Comment:
Please also add Chen21e-B.1.351, Chen21e-B.1.1.28, Chen21e-B.1.429 and Chen21e-B.1.617.1.

Reply:
I have pushed changes to my forked repository

Reply:
Thank you!